packages/framer-motion/src/motion/index.tsx

```diff
@@ -143,7 +145,43 @@ export function createMotionComponent<Props extends {}, Instance, RenderState>({
         )
     }
 
-    return forwardRef(MotionComponent)
+    const ForwardRefComponent = forwardRef(MotionComponent)
+
+    Object.defineProperty(ForwardRefComponent, wrappedComponentSymbol, {
```

Comment:
Not sure if we should do this instead:
```
ForwardRefComponent[wrappedComponentSymbol] = Component
```

Reply:
Looks like `enumerable: false` is not doing anything for symbols. Not sure if we need to make this readonly. Setting the property directly would make this code more readable.

Reply:
Yeah I'd just set it directly